includes/admin/promos/notices/class-five-star-review-dashboard.php

```diff
@@ -0,0 +1,193 @@
+		</div>
+		<div class="edd-review-step edd-review-step-3" style="display:none;">
+			<p><?php esc_html_e( 'That\'s awesome! Could you please do me a BIG favor and give it a 5-star rating on WordPress to help us spread the word and boost our motivation?', 'easy-digital-downloads' ); ?></p>
+			<p><strong><?php echo wp_kses( __( '~ Chris Klosowski<br>President of Easy Digital Downloads', 'easy-digital-downloads' ), array( 'br' => array() ) ); ?></strong></p>
+			<div class="edd-review-actions">
+				<a href="https://wordpress.org/support/plugin/easy-digital-downloads/reviews/?filter=5#new-post" class="button button-primary edd-promo-notice-dismiss" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'Ok, you deserve it!', 'easy-digital-downloads' ); ?></a><br>
+				<button class="button-link edd-promo-notice-dismiss"><?php esc_html_e( 'No thanks', 'easy-digital-downloads' ); ?></button>
+			</div>
+		</div>
+		<img alt="" class="edd-peeking" src="<?php echo esc_url( EDD_PLUGIN_URL . 'assets/images/edd-peeking.png' ); ?>" />
+		<script type="text/javascript">
+			document.addEventListener( 'DOMContentLoaded', function() {
+				var steps = document.querySelectorAll( '.edd-review-switch-step' );
+				steps.forEach( function(step) {
+					step.addEventListener( 'click', function ( e ) {
+						e.preventDefault();
+						var target = this.getAttribute( 'data-step' );
+						if ( target ) {
+							var notice = this.closest( '.edd-promo-notice' );
+							var review_step = notice.querySelector( '.edd-review-step-' + target );
+							if ( review_step ) {
+								var thisStep = this.closest( '.edd-review-step' );
+								eddFadeOut( thisStep );
+								eddFadeIn( review_step );
+							}
+						}
+					} )
+				} )
+
+				function eddFadeIn( element ) {
+					var op = 0;
+					element.style.opacity = op;
+					element.style.display = 'block';
+					var timer = setInterval( function () {
+						if ( op >= 1 ) {
+							clearInterval( timer );
+						}
+						element.style.opacity = op;
+						element.style.filter = 'alpha(opacity=' + op * 100 + ')';
+						op = op + 0.1;
+					}, 80 );
+				}
+
+				function eddFadeOut( element ) {
+					var op = 1;
+					var timer = setInterval( function () {
+						if ( op <= 0 ) {
+							element.style.display = 'none';
+							clearInterval( timer );
+						}
+						element.style.opacity = op;
+						element.style.filter = 'alpha(opacity=' + op * 100 + ')';
+						op = op - 0.1;
+					}, 80 );
+				}
+			} );
+		</script>
+		<?php
+	}
+
+	/**
+	 * Whether the notice should display.
+	 *
+	 * @since 2.11.4
+	 * @return bool
+	 */
+	protected function _should_display() {
+
+		$activated = edd_get_activation_date();
+		if ( $activated ) {
+			if ( ( $activated + ( DAY_IN_SECONDS * 30 ) ) > time() ) {
+				return false;
+			}
+		}
+		// @todo Change this to edd_count_orders in 3.0
+		$payments = edd_count_payments();
+		if ( $payments && 15 > $payments->publish ) {
+			return false;
+		}
+
+		return true;
```

Comment:
```suggestion
		return isset( $payments->publish ) && $payments->publish >= 15;
```

I went through several other suggestions before ending up on this one lol.

Originally I was going to change the `if` you have now to:

```
if ( isset( $payments->publish ) && 15 > $payments->publish ) {
```

But then I realized it probably makes more sense to return `false` if `$payments->publish` isn't set for some reason... Then ultimately I realized it feels a bit better to me visually for the `if` check to be looking for a successful return, which led to:

```
if ( isset( $payments->publish ) && $payments->publish >= 15 )
```

Then simplified it. :stuck_out_tongue: